tests/MercadoPago/Client/Cardtoken/CardTokenClientUnitTest.php

```diff
@@ -7,53 +7,53 @@
 use PHPUnit\Framework\TestCase;
 
 /**
- * PaymentClient unit tests.
+ * CardTokenClient unit tests.
  */
 final class CardTokenClientUnitTest extends TestCase
 {
     public function testCreateSuccess(): void
     {
-        $filepath = '../../../Resources/Mocks/Response/Cardtoken/card_token_get.json';
-        $mockHttpRequest = $this->mockHttpRequest($filepath, 200);
+        $file_path = '../../../Resources/Mocks/Response/Cardtoken/card_token_get.json';
```

Comment:
Ajustar nome da pasta para CardToken
```suggestion
        $file_path = '../../../Resources/Mocks/Response/CardToken/card_token_get.json';
```